maintenance/scripts/setup.sh

```diff
@@ -1,5 +1,30 @@
 #! /bin/bash
 
+Help()
+{
+  echo "Setup the project and your environment."
+  echo
+  echo "Syntax: setup.sh [-h|c]"
+  echo "   -h   Show this help message."
+  echo "   -c   Setup the environment for CI."
+  echo
+}
+
+CI=false
+
+while getopts ":hc" option; do
+  case $option in
+    h | help) # display Help
+      Help
+      exit;;
+    c | ci)
+      CI=true;;
```

Comment:
I think we should prefix these internal vars with our own prefix to avoid collision with envvars that might be set outside script. For example, CI=true is always set in GitHub Actions and many CI environments:
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

This might drive someone bonkers if they ever want to unset this on CI for some sanity check or something :)

Reply:
Ah wait, I'm seeing that `CI` system value is being overridden with `false` before, but honestly, best to just use a prefix and not step on the toes of the CI environment, because CI is the var used in every system since TravisCI. (and we're not using this envvar as it was intended to detect system automatically, but rather as a var to store a script flag)

Reply:
Thanks, I was not aware that  github actions already used that var. Prefixed in [prefix CI var](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6708/commits/90dcbb3290d091c60d6e25ee11ab62505eb22cbd)